docs/LinksAndButtons.md

```diff
@@ -0,0 +1,97 @@
+* iOS VoiceOver has the same verbal settings and feel as Desktop VoiceOver, lending it a familiarity to Apple desktop users. Recommend future testing include mobile VoiceOver testing when possible.
+
+### Unique observations from our testers
+
+* One user was looking for a link to jump to a form farther down the page. Could be an anchor ID to a containing div, if the form is farther down the page from a descriptive block of text.
+* If the call to action, form, etc. is on the current page, use a link with descriptive “further down this page” language
+* If the call to action, form, etc, is on another page, use a link with descriptive “another page” language
+
+## Research findings <sup id="heading2">[2]</sup>
+
+It was the consensus of our test users that call to action links can be styled as buttons. The underlying HTML is an `<a>` tag that must meet the success criteria outlined below, and visually it will look like a button. Sighted users rely on this visual information, so having the call to action presented in a strong, clear manner is important. Screen reader users rely more on semantic meaning. Calls to action need to offer clear language about what we are asking users to do. If calls to action have short text blocks, say 2-4 words, consider adding a more descriptive aria-label for assistive devices.
+
+### Example link with aria-label for semantic meaning
+
+```html
+  <a
+    href=“#”
+    aria-label=“Start your application for health coverage on the next page”
+  >
+    Start your application
+  </a>
+```
+
+### Benefits for assistive devices
+
+* Sighted users will interpret “Start your application” and have the heading, sub-heading, and content around the CTA to determine intent
+* Screen reader users will hear “Start your application for health coverage on the next page” read out, and also have context because of the aria-label
+
+## Three rules for link and button success <sup id="heading3">[3]</sup>
+
+1. Links go to to another page OR jump to an anchor ID in the current page
+    1. SUCCESS CRITERIA:
+    2. Respond to ENTER keypress
+    3. The browser URL changes
+    4. HTML5 history object changes
+    5. Avoid hash route URLs if possible. These interfere with in-page anchor links, and were common with early SPA libraries like Backbone.
+    6. Append anchor links to URL when a link points to an in-page anchor ID (skip nav, tabs)
+    7. Do not add role=“button” to links. Screen readers announce the link as “button”, and changes user   * Recommended to underline links when in paragraphs or blocks of text
```

Comment:
Is the asterisk in here intentional?

Reply:
@jbalboni Nope, just missed it in reformatting. I'll fix before merging.